pkg/client/images.go

```diff
@@ -61,3 +62,10 @@ func (c *images) Delete(name string) (err error) {
 	err = c.r.Delete().Resource("images").Name(name).Do().Error()
 	return
 }
+
+// UpdateSignatures allows to modify signatures of given image.
+func (c *images) UpdateSignatures(image *imageapi.Image) (result *imageapi.Image, err error) {
```

Comment:
> NamedCreater.  I don't know if someone removed the NamedUpdater equivalent
> or not.

@miminar that's the one you want.  And since its a different type, POST works out very cleanly.

As you expand the storage, I'd like to see a stub switch/case calling out known options for interpreting the signature.